test/README.md

```diff
@@ -250,3 +250,9 @@ Content-Type: application/json
 
 	OK
 	```
+## Strictness levels
```

Comment:
I'd suggest to call it tags or categories. So there might be intersections. Like forward-only mode tests and strict mode the same time.

Reply:
Yes, categories would also be an option. Let's do that in another PR.